charts/gateway-helm/templates/certgen-rbac.yaml

```diff
@@ -62,3 +62,63 @@ subjects:
 - kind: ServiceAccount
   name: '{{ include "eg.fullname" . }}-certgen'
   namespace: '{{ .Release.Namespace }}'
+---
+{{- if .Values.topologyInjector.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: '{{ include "eg.fullname" . }}-certgen:{{ .Release.Namespace }}'
```

Comment:
do we really need to append the namespace in the suffix ?
we dont do this for cert-gen or envoy-gateway deployments since they are unique to a namespce

Reply:
oh its cluster scoped, nvm

Reply:
nvm we need this

Reply:
Yes but as you said those are already unique to a namespace whereas cluster roles aren't. In order to support namespaced deployments we'll need unique ClusterRoles per-install.